pallets/ethereum-checked/src/benchmarking.rs

```diff
@@ -18,7 +18,7 @@
 
 use super::*;
 
-use astar_primitives::ethereum_checked::MAX_ETHEREUM_TX_INPUT_SIZE;
+use astar_primitives::ethereum_checked::EthereumTxInput;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `EthereumTxInput` is missing. Please add the import statement for `EthereumTxInput` at line 21.
```diff
+use astar_primitives::ethereum_checked::EthereumTxInput;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->